build_resolvers/pubspec.yaml

```diff
@@ -9,11 +9,15 @@ environment:
 
```

Comment:
Need to update the version also (0.2.3-dev?)

Same for build_runner/build_runner_core (and update changelogs for those)